tests/components/ShareForm/ShareForm.test.jsx

```diff
@@ -0,0 +1,77 @@
+import { render, screen } from '@testing-library/react';
+import ShareForm from '../../../src/components/ShareForm';
+import * as FirebaseFunctions from '../../../src/api/firebase';
+import { vi } from 'vitest';
+import { QueryClientProvider } from 'react-query';
+import { queryClient } from '../../../src/App';
+import userEvent from '@testing-library/user-event';
+
+const sleep = async (ms = 200) =>
```

Comment:
nit: This is because of me, but would be nice if this was moved into a test util instead so it's not just copied an pasted into each test

Reply:
will do. also i think we went over this but just wanted to ask again so it solidifies in my brain. we are calling the sleep method to mimic the delay that a firebase function would make, right? @bdanzer

Reply:
@jtkabenni That's correct!